easybuild/easyconfigs/b/bokeh/bokeh-1.3.4-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'PythonBundle'
+
+name = 'bokeh'
+version = '1.3.4'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://github.com/bokeh/bokeh'
+description = "Statistical and novel interactive HTML plots for Python"
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+dependencies = [
+    ('Python', '3.7.2'),
+    ('PyYAML', '5.1'),
+    ('Pillow', '6.0.0'),
+    ('SciPy-bundle', '2019.03'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('MarkupSafe', '1.1.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/m/MarkupSafe/'],
+        'checksums': ['29872e92839765e546828bb7754a68c418d927cd064fd4708fab9fe9c8bb116b'],
+    }),
+    ('Jinja2', '2.10.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/J/Jinja2'],
+        'checksums': ['065c4f02ebe7f7cf559e49ee5a95fb800a9e4528727aec6f24402a5374c65013'],
+    }),
+    ('tornado', '6.0.3', {
+        'source_urls': ['https://pypi.python.org/packages/source/t/tornado'],
+        'checksums': ['c845db36ba616912074c5b1ee897f8e0124df269468f25e4fe21fe72f6edd7a9'],
+    }),
+    ('packaging', '19.1', {
```

Comment:
@branfosj `MarkupSafe`, `Jinja2` and `packaging` are already included with Python 3.7.2, so you can drop these from `exts_list` (you'll still need `tornado` + `bokeh` itself)

Reply:
done